tests/php/sync/test_class.jetpack-sync-callables.php

```diff
@@ -1216,25 +1216,23 @@ public function test_sync_does_not_send_updates_if_array_order_is_only_change()
 	}
 
 	/**
-	 * Randomize the get_plugins array keys.
+	 * Reorder the get_plugins array keys.
 	 *
 	 * @param array $plugins array of plugins.
 	 *
 	 * @return array
 	 */
-	public function randomize_array_keys( $plugins ) {
+	public function reverse_array_keys( $plugins ) {
```

Comment:
You called it "reorder_array_keys" above, but "reverse_array_keys" here.
```suggestion
	public function reorder_array_keys( $plugins ) {
```